### PR TITLE
Triplets  acceptance radius

### PR DESCRIPTION
**Pull request by mrivi**
**Describe problem solved by the proposed pull request**
The `acceptance_radius` field of a triplet is set only when explicitly setting param2 in the mission item (see https://github.com/PX4/Firmware/pull/10413#discussion_r218913242). When using the default `NAV_ACC_RAD`, the field in the triplet is always `0.0`.
This PR sets the triplet  `acceptance_radius` to the default `NAV_ACC_RAD` unless the mission item has a param2 different from zero. The triplet field is checked in the Auto FlightTasks instead of the parameter. 

**Test data / coverage**
Only tested on SITL.

### CODE REVIEW

**Comment by dagar** [comment]
It's not clear to me how acceptance radius not set is supposed to be expressed. This is the type of thing that's unclear in the mavlink spec. At the moment I'm guessing 0 or NAN could both be used.

image

https://mavlink.io/en/messages/common.html#MAV_CMD

@DonLakeFlyer could you provide any insight here?

**Comment by DonLakeFlyer** [comment]
QGC sets to 0 to mean not set.

**Comment by mrivi** [comment]
@dagar Right now, param2 is zero when not set

File: src/modules/navigator/mission_block.cpp [hunk 1]
```diff
@@ -508,7 +508,12 @@ MissionBlock::mission_item_to_position_setpoint(const mission_item_s &item, posi
 	sp->loiter_radius = (fabsf(item.loiter_radius) > NAV_EPSILON_POSITION) ? fabsf(item.loiter_radius) :
 			    _navigator->get_loiter_radius();
 	sp->loiter_direction = (item.loiter_radius > 0) ? 1 : -1;
-	sp->acceptance_radius = item.acceptance_radius;
+	sp->acceptance_radius = _navigator->get_default_acceptance_radius();
+
+	if (item.acceptance_radius > 0.0f) {
```
Comment (dagar) [inline src/modules/navigator/mission_block.cpp]: To be paranoid (until it's documented) could you check ISFINITE as well? Setting the default in else would also be slightly better (only touching sp->acceptance_radius once).

**Comment by sanderux** [comment]
careful: VTOL_LAND sets a dynamic acceptance radius on the setpoint.

**Comment by mrivi** [comment]
@sanderux could you please point me to where the acceptance radius is dynamically changed?

**Comment by dagar** [comment]
@sanderux we need to fix and review that for VTOL regardless.

**Pull request merged**